skl2onnx/operator_converters/gaussian_process.py

```diff
@@ -0,0 +1,126 @@
+# -------------------------------------------------------------------------
```

Comment:
Could you add unit tests to improve coverage for this file? It's just 18%.
skl2onnx/operator_converters/gaussian_process.py              56     46    18%   15-16, 36-121

Reply:
Same answer as above.